Lib/test/test_email/test_email.py

```diff
@@ -2261,50 +2261,6 @@ def test_multipart_no_boundary(self):
         self.assertIsInstance(msg.defects[1],
                               errors.MultipartInvariantViolationDefect)
 
```

Comment:
The test methods `test_same_boundary_inner_outer` and `test_mulltipart_no_boundary` are now also redundant and can be deleted.